promscale/manage-data/index.md

```diff
@@ -0,0 +1,14 @@
+# Manage data in Promscale
+Promscale provides different methods to manage the data you store in Promscale
+and help you reduce disk space and improve query performance.
```

Comment:
"reduce disk space" - shouldn't this be "reduce disk space usage"?

Reply:
```suggestion
and help you reduce disk space use and improve query performance.
```